locales/th/common.json

```diff
@@ -1,4 +1,11 @@
 {
   "signIn": "เข้าสู่ระบบ",
-  "signUp": "สมัครสมาชิก"
+  "signUp": "สมัครสมาชิก",
+  "helpCenter": "ศูนย์ให้บริการความช่วยเหลือ",
+  "termsOfService": "เงื่อนไขการให้บริการ",
+  "privacyPolicy": "นโยบายความเป็นส่วนตัว",
+  "emergencyLine": "สายฉุกเฉิน",
```

Comment:
Since we don't have any contact number yet, I agree to change it to "Helpline" for now. Are you okay if I change `font-color` to `black` as others in the section.